drivers/lora/sx12xx_common.c

```diff
@@ -18,15 +18,15 @@
 LOG_MODULE_REGISTER(sx12xx_common, CONFIG_LORA_LOG_LEVEL);
 
 static struct sx12xx_data {
-	struct k_sem data_sem;
+	struct k_poll_signal *operation_done;
```

Comment:
Perhaps name this as `rx_sig`? As `operation` is too generic.

Reply:
The same signal is used in #36577, hence the generic name rather than introducing it here then renaming it in the next PR.

Reply:
`operation_done` is reused there also. So I don't see any issue in renaming it here and in #36577.

Reply:
I don't understand. Yes it is reused there. At that point it is used to signal both TX and RX completion. Why do you want it renamed in both PR's?

Reply:
Ah, didn't see that it is used for both Rx/Tx. Sorry for the confusion.